common/lc_colors.cpp

```diff
@@ -222,22 +222,21 @@ static void lcAdjustStudStyleColors(std::vector<lcColor>& Colors, lcStudStyle St
 	const lcVector4 BlackEdge = lcVector4FromColor(Preferences.mBlackEdgeColor);
 
 	const float ContrastControl = Preferences.mPartEdgeContrast;
-	const float LightDarkControl = Preferences.mAutomateEdgeColor ?
-				Preferences.mPartColorValueLDIndex :
-				LC_SRGB_TO_LINEAR(Preferences.mPartColorValueLDIndex);
+	const float LightDarkControl = LC_SRGB_TO_LINEAR(Preferences.mAutomateEdgeColor ? Preferences.mPartColorValueLDIndex : Preferences.mPartColorValueLDIndex);
```

Comment:
This particular hunk needs to be reverted; the old code was correct. See my note in the #621 thread for the reason it was done that way.

The other changes appear to be correct (in this first reading).